teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,19 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val beforeRevision: String?
```

Comment:
What is stored in `before/afterRevision` properties? Do they store full content of file or just the revision number (hash)? If they store full content, I think it makes sense to rename them to something like `oldContent`/`newContent`.